app/src/main/java/org/sopt/and/SignInActivity.kt

```diff
@@ -0,0 +1,204 @@
+import androidx.compose.material3.CenterAlignedTopAppBar
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.IconButton
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.SnackbarHost
+import androidx.compose.material3.SnackbarHostState
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.rememberCoroutineScope
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.RectangleShape
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.launch
+import org.sopt.and.components.InputField
+import org.sopt.and.components.PasswordInputField
+import org.sopt.and.components.SocialLoginRow
+import org.sopt.and.ui.theme.ANDANDROIDTheme
+import java.io.Console
+
+class SignInActivity : ComponentActivity() {
+    private var savedEmail: String? = null
+    private var savedPassword: String? = null
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        savedEmail = intent.getStringExtra("EMAIL")
+        savedPassword = intent.getStringExtra("PASSWORD")
+
+        enableEdgeToEdge()
+        setContent {
+            ANDANDROIDTheme(true) {
+                SignInScreen(savedEmail, savedPassword)
+            }
+        }
+    }
+}
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun SignInScreen(
+    savedEmail: String?,
+    savedPassword: String?
+){
+    var email by remember { mutableStateOf("") }
+    var password by remember { mutableStateOf("") }
+    var passwordVisible by remember { mutableStateOf(false) }
+    val context = LocalContext.current
+    val snackbarHostState = remember { SnackbarHostState() }
+    val coroutineScope = rememberCoroutineScope()
+
+    fun onSignInClick() {
+        coroutineScope.launch {
+            Log.d("11", email + " " + savedEmail + " " + password + " " + savedPassword)
+            if(email == savedEmail && password == savedPassword) {
+                snackbarHostState.showSnackbar("로그인에 성공했습니다.")
+                delay(1L)
+                val intent = Intent(context, MyActivity::class.java).apply {
+                    putExtra("EMAIL", email)
+                    putExtra("PASSWORD", password)
+                }
+                context.startActivity(intent)
```

Comment:
also라는 스코프함수도 있습니다 !
```suggestion
                val intent = Intent(context, MyActivity::class.java).apply {
                    putExtra("EMAIL", email)
                    putExtra("PASSWORD", password)
                }.also { context.startActivity(intent) }
```